src/api/users/README.md

```diff
@@ -61,7 +61,9 @@ By default the server is running on <http://localhost:6666/>.
 
 \- `GET /:id` - returns 200 with the user specified by the id, or 404 if a user does not exist.
 
-\- `GET /` - returns 200 with all Telescope users in an array, or 404 if the `users` collection is empty.
+\- `GET /` - returns 200 with the first 5 Telescope users (numerically sorted by `id`) in an array, or 404 if the `users` collection is empty.
```

Comment:
Consolidate this line and the next into one, since `/` is the same as `/?per_page=10&page=1`, it just defines default values.

Also, I think 5 is the wrong default.  Something like 10 or 20 seems better (i.e., if I was using this in a web page, I'd always want to show more than 5 at once, but probably not more than 20).